watcher/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM lsstts/develop-env:sal_v4.0.0_salobj_v5.0.0
+FROM lsstts/develop-env:b32
```

Comment:
Everything else is using b42. What's in b32?

Reply:
The changes on the containers above where mainly on the config package.